packages/calcite-components/src/components/action-group/action-group.scss

```diff
@@ -54,15 +61,18 @@
 
 :host([layout="grid"]) {
   @apply grid;
-}
 
-:host([layout="grid"]) .container {
-  @apply bg-background
-  grid
-  place-content-stretch;
-  gap: var(--calcite-action-group-gap);
-  padding: var(--calcite-action-group-gap);
-  grid-template-columns: repeat(var(--calcite-action-group-columns), auto);
+  .container {
+    @apply grid
+    place-content-stretch;
+    background-color: var(--calcite-action-background-color, var(--calcite-color-foreground-1));
+    gap: var(--calcite-action-group-gap, theme("gap.px"));
+    padding: var(--calcite-action-group-gap, theme("gap.px"));
```

Comment:
Not sure we'd want to allow these to be changed in the current set up where `layout` isn't set on the Action Group.

Reply:
I moved some code around for organizational purposes but this was the previous default. Gap won't do anything unless flex is set which is only added via a `layout`

Reply:
Cool, maybe we can have a note to remove it when the "layout" is removed (it is currently deprecated)

Reply:
Added a [Deprecated] to the `--calcite-action-group-gap` docs. Should I add a note to remove the token anywhere else?

Reply:
That should be good 👍